cypress/support/bot.ts

```diff
@@ -17,6 +17,9 @@ limitations under the License.
 /// <reference types="cypress" />
 
 import request from "browser-request";
+import { MatrixScheduler, MemoryStore, MemoryCryptoStore } from "matrix-js-sdk/src/matrix";
+import { WebStorageSessionStore } from "matrix-js-sdk/src/store/session/webstorage";
+import { MockStorageApi } from "matrix-js-sdk/spec/MockStorageApi";
```

Comment:
These all need taking from `matrixcs` see https://github.com/matrix-org/matrix-react-sdk/blob/develop/docs/cypress.md#using-matrix-js-sdk again

Reply:
`MockStorageApi` isn't part of `matrixcs`, should it be?

Reply:
Ah right, it shouldn't be, and also won't be importable in CI due to it being an environment operating on the webpack output of an element-web build.